packages/ui/src/Layout/Layout.tsx

```diff
@@ -1,15 +1,28 @@
 import React from "react";
 
-import { EisbukLogo } from "@eisbuk/svg";
+import { EisbukLogo, People, LibraryBooks, Calendar } from "@eisbuk/svg";
 
 import UserAvatar, { UserAvatarProps } from "../UserAvatar";
+import AdminBar from "./AdminBar";
+import { NavigationLabel, useTranslation } from "@eisbuk/translations";
 
 interface LayoutProps {
   Logo?: React.FC;
   user?: UserAvatarProps;
   additionalButtons?: JSX.Element;
   Notifications?: React.FC<{ className?: string }>;
   children?: React.ReactNode[] | React.ReactNode;
+  isAdmin: boolean;
+}
+interface Item {
+  Icon: string;
+  label: string;
+  slug: string;
+}
+enum PrivateRoutes {
```

Comment:
This shouldn’t be hardcoded and is an app context (should be dynamically passed to the layout component from inside the app)